### DISCUSSION TITLE
Flows - How to define retry logic when a condition is not met?

### DISCUSSION THREAD

**Discussion by DeanKamali** [directus/directus#13872] [Archive]
Hey everyone, 

I want to repeat a step when a "condition" is not met; I haven't been able to re-wire the steps; is that even possible? If not, how can I go about defining such logic? 

Thanks

**Comment by rijkvanzanten** [comment]
There's currently no way to setup a circular operation chain, to prevent infinite loops. That being said, you can use the "trigger another flow" operation to trigger another flow when the condition is not met

**Reply by DeanKamali** [reply]
Make sense, I will try that, thank you
